src/JustSaying/Fluent/AccountAddressProvider.cs

```diff
@@ -109,12 +106,8 @@ public Uri GetQueueUriByConvention<T>()
     /// <returns>The <see cref="Uri"/> for this queue.</returns>
     public Uri GetQueueUri(string queueName)
     {
-        var endpoint = new AmazonSQSEndpointProvider().ResolveEndpoint(new SQSEndpointParameters
-        {
-            Region = _regionEndpoint.SystemName
-        });
-
-        return new UriBuilder(endpoint.URL)
+        var hostname = _regionEndpoint.GetEndpointForService("sqs").Hostname;
+        return new UriBuilder("https", hostname)
```

Comment:
https://learn.microsoft.com/dotnet/api/system.uri.urischemehttps

```suggestion
        return new UriBuilder(Uri.UriSchemeHttps, hostname)
```